packages/experimental/Spinner/src/mobile/Spinner.types.tsx

```diff
@@ -0,0 +1,50 @@
+import { Animated, ColorValue } from 'react-native';
+import { SvgProps } from 'react-native-svg';
+import { ActivityIndicatorProps as CoreActivityIndicatorProps } from 'react-native';
+
+export const spinnerName = 'Spinner';
+/**
+ * Specifies the possible sizes of the Spinner.
+ */
+export type SpinnerSize = 'xSmall' | 'small' | 'medium' | 'large' | 'xLarge';
+
+export interface SpinnerTokens {
+  /**
+   * Spinner element color
+   * @defaultValue 'grey56' for light mode, 'grey72' for dark mode
+   */
+  spinnerColor?: ColorValue;
+  /**
+   * Line thickness of the Spinner
```

Comment:
Can we keep the types as only "`ts`" and export out of package accordingly. It would also be more consistent if there was no specific mobile folder that defines its own types. The `SpinnerProps` here are already added in `/src/Spinner.types.tsx`. In general guidance, the types is okay being a superset, and same applies to other definitions in this file. Maybe choose an [intersection type interface](https://www.typescriptlang.org/docs/handbook/2/objects.html#intersection-types)  to make the code cleaner while defining the superset types.

Reply:
yes working towards that will bring everything under one types file.

Reply:
Brought into common types files. Added approach in description.